optest/optest_test.go

```diff
@@ -221,3 +254,31 @@ func testGenerateState(t *testing.T) string {
 
 	return state
 }
+
+func TestNew(t *testing.T) {
+	testUsers := map[string]TestUser{
+		"foo": {
+			Audience:           "foo-client",
+			Subject:            "foo",
+			Name:               "Foo Bar",
+			GivenName:          "Foo",
+			FamilyName:         "Bar",
+			Locale:             "en-US",
+			Email:              "foo@bar.com",
+			AccessTokenKeyType: "JWT+AT",
+			IdTokenKeyType:     "JWT",
+		},
+	}
+
+	_, err := New()
+	require.NoError(t, err)
+
+	_, err = New(WithTestUsers(nil))
```

Comment:
I think this should test with an empty map rather than nil, because that's the more expected "wrong way" of doing it.

(or possibly test both)